src/containers/ProjectDashboardContainer.jsx

```diff
@@ -43,9 +44,11 @@ class ProjectDashboardContainer extends Component {
   }
 
   handleSelect({ option }) {
+    const { actions } = this.props;
     this.setState({
       option,
     });
+    actions.createTranslation('project_contents', option.value);
```

Comment:
I think `createTranslation` can perhaps be dropped. From the front-end, you're either selecting a translation to edit, or updating the selected translation. I don't think you ever create a translation directly.